docs/user_guide/simulator/simulator.rst

```diff
@@ -231,17 +259,34 @@ number of octants is deciphered form the value of the *integration_volume* attri
 The default value of this attribute, 70, produces 2556 orientations at which the NMR
 frequency contribution is evaluated.
 
-.. code-block:: python
+.. plot::
+    :context: close-figs
 
     sim = Simulator()
-    print(sim.config.integration_density)
-    # 70
+    print(sim.config.integration_density)  # default
+
+.. plot::
+
+    70
+
```

Comment:
Why do you use plot for this?

Reply:
It is just used as rendered code to display the output of printing ``sim.config.integration_density``. Not sure of an easer way to display code output when using code-block / plot.

Reply:
The second plots block is not testing the output of the first plot block. Maybe leave it as a comment as before!